### PR TITLE
DM-46089: Add mosaic view for AuxTel camera

### PR DESCRIPTION
**Pull request by ugyballoons**
The view can be accessed via the relative link [rubintv/tucson-usdf/auxtel/mosaic](url)

### CODE REVIEW

File: python/lsst/ts/rubintv/handlers/websocket_notifiers.py [hunk 1]
```diff
@@ -11,45 +11,51 @@
...
-    clients_list: list[UUID], data_type: str, payload: Mapping
+    clients_list: list[UUID],
+    service: Service,
+    message_type: MessageType,
+    payload: Mapping,
 ) -> None:
     tasks = []
     async with clients_lock:
-        logger.info("Sending updates to:", num_clients=len(clients_list))
+        # logger.info("Sending updates to:", num_clients=len(clients_list))
```
Comment (sebastian-aranda) [inline python/lsst/ts/rubintv/handlers/websocket_notifiers.py:36]: Should this comment be removed?

**Review by sebastian-aranda — CHANGES_REQUESTED** [review]
Sorry it took me so long to finish this review, I got distracted with several things in the way :dizzy_face:. Thanks for the efforts put here @ugyballoons , I left some comments, mostly suggestion, but anyways I ask you to take a look before merging please.

File: python/lsst/ts/rubintv/models/models.py [hunk 2]
```diff
@@ -70,55 +70,20 @@ class HasButton(BaseModel):
...
 
 
-class Camera(HasButton):
-    """Represents a camera entity, capable of handling different channels like
-    images or movies.
+class MosaicViewMeta(BaseModel):
+    channel: str
+    metaColumns: list[str]
 
-    This class extends the Pydantic BaseModel to leverage data validation. It
```
[the remark this answers is not in the document]
Comment (ugyballoons) [reply python/lsst/ts/rubintv/models/models.py:77]: Oops! That wasn't deliberate. I've replaced it and added the extra MosaicViewMeta property to the docs.

File: python/lsst/ts/rubintv/background/currentpoller.py [hunk 3]
```diff
@@ -129,7 +130,7 @@ async def poll_buckets_for_todays_data(self, test_day: str = "") -> None:
                         break
 
                 elapsed = time() - timer_start
-                logger.info("Current - time taken:", elapsed=elapsed)
+                # logger.info("Current - time taken:", elapsed=elapsed)
```
[the remark this answers is not in the document]
Comment (ugyballoons) [reply python/lsst/ts/rubintv/background/currentpoller.py:133]: I've restored the logging

File: python/lsst/ts/rubintv/handlers/websocket.py [hunk 4]
```diff
@@ -99,6 +67,19 @@ async def data_websocket(
         logger.exception("Caught surprise exception")
 
 
+async def validate_raw_message(raw: str) -> tuple[uuid.UUID, dict] | None:
+    try:
+        data: dict = json.loads(raw)
+    except json.JSONDecodeError as e:
+        logger.error("JSON not well formed", error=e)
+        return None
```
[the remark this answers is not in the document]
Comment (ugyballoons) [reply python/lsst/ts/rubintv/handlers/websocket.py:75]: I looked this up as I understand one of the tenets of Python is `Explicit is better than implicit.`
[PEP-8](https://peps.python.org/pep-0008/#programming-recommendations) suggests not mixing `return` and `return something` in the same function, so I'll keep them in.

**Pull request merged**